roachpb/api.proto

```diff
@@ -236,8 +236,10 @@ message EndTransactionResponse {
   optional ResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
   // Remaining time (ns).
   optional int64 commit_wait = 2 [(gogoproto.nullable) = false]; // TODO(tschottdorf): remove this
-  // List of intents resolved by EndTransaction call.
-  repeated bytes resolved = 3 [(gogoproto.casttype) = "Key"];
+  // Obsolete tag. Removed in #5966.
+  repeated bytes obsolete_resolved = 3;
```

Comment:
(Reviewable is being annoying, so back to github)

We don't have a style for obsolete or deprecate tags, but I'd prefer something that is louder: `OBSOLETE_removed` or `DEPRECATED_removed`.